spoc/contacts.py

```diff
@@ -1,15 +1,22 @@
 """Managing multi-way contacts."""
-
 from __future__ import annotations  # needed for self reference in type hints
-from itertools import permutations, product
-from typing import List, Optional, Dict
-import pandas as pd
+
+from itertools import permutations
```

Comment:
It's done by reorder-python-imports in the pre-commit hooks. It separates builtin from third-party from imports in this repo. It's very handy because it reorders imports automatically :)

Reply:
Ah, I meant that before there were multiple objects imported per line (List, Optional, Dict), and now they are imported in individual lines &mdash; but if that's an automatic behaviour, then it doesn't matter :)